cluster/azure/pod_identity/variables.tf

```diff
@@ -0,0 +1,43 @@
+variable "enable_pod_identity" {
+  type    = "string"
+  default = "true"
+}
+
+variable "resource_group_name" {
+  type = "string"
+}
+
+variable "subscription_id" {
+  type = "string"
+}
+
+variable "pod_identity_install_url" {
+  type = "string"
+  default = "https://raw.githubusercontent.com/Azure/aad-pod-identity/1.3.0-mic-1.4.0-nmi/deploy/infra/deployment-rbac.yaml"
+}
```

Comment:
Any specific reason for not using the master url? `https://raw.githubusercontent.com/Azure/aad-pod-identity/master/deploy/infra/deployment-rbac.yaml` Using the `1.3.0-mic...` url I ran into [this issue](https://github.com/Azure/kubernetes-keyvault-flexvol/issues/120).